app/views/shared/_client_primary_info_fields.html.erb

```diff
@@ -1,84 +1,51 @@
 <% is_dropoff = is_dropoff || false %>
+<% is_ctc = is_ctc || false %>
 <div id="primary-info">
-  <h3><%= t("hub.clients.fields.primary_contact_info") %>:</h3>
-  <div class="hub-form__row fill">
+  <div class="hub-form__card card-small">
+    <h3><%= t("hub.clients.fields.primary_contact_info") %></h3>
     <%= f.cfa_input_field(:primary_first_name, t("hub.clients.fields.legal_first_name")) %>
     <%= f.cfa_input_field(:primary_last_name, t("hub.clients.fields.legal_last_name")) %>
-  </div>
-  <%= f.cfa_input_field(:preferred_name, t("hub.clients.fields.preferred_name")) %>
+    <%= f.cfa_input_field(:preferred_name, t("hub.clients.fields.preferred_name")) %>
 
-  <% if is_dropoff %>
-    <%= f.cfa_select(:timezone, t("general.timezone"), timezone_select_options, selected: current_user.timezone) %>
-  <% else %>
-    <%= f.cfa_select(:timezone, t("general.timezone"), timezone_select_options) %>
-  <% end %>
+    <% if is_ctc %>
```

Comment:
If the form continues to diverge, I think we might want to think about a different pattern. However, I'm planning on asking design if we can swap out the regular drop_off creation with a hidden field too since there are very low chances that the person coming into a physical office will use a different time zone than the user creating their account.